benchmarks-next/run.sh

```diff
@@ -5,11 +5,33 @@ PROJECT_DIR=$(dirname "$0" | xargs dirname)
 SIGHTGLASS="cargo +nightly run --bin sightglass-cli --"
 export RUST_LOG=debug
 
-$SIGHTGLASS build-engine wasmtime
+ENGINE=${ENGINE:-wasmtime}
```

Comment:
`run.sh` is a temporary placeholder, and any improvements to it, should instead be made to the benchmark runner CLI itself.

Reply:
Well, this was useful to run a bunch of wasm modules, but I'll remove this commit then.